project1/index.html

```diff
@@ -6,67 +6,66 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="styles.css">
-    <title>Document</title>
+    <title>Smart Octopus</title>
 </head>
 
 <body>
-
-    <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
-
-    <h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
-        intelligent.</h2>
-
-    <figure>
-        <img src="images/merlin.jpg" alt="lignt pink octopus">
-        <figcaption>An octopus in an aquarium in Brest, France. The animals are highly intelligent, but researchers are
-            uncertain how the trait evolved</figcaption>
+    <h1>Yes,
+        the Octopus Is Smart as Heck. But Why?</h1>
+    <h2>It has eight arms,
+        three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so intelligent.</h2>
+    <figure><img src="images/octopus.jpg" alt="octopus illustration">
+        <figcaption>An purple octopus illustration. The animals are highly intelligent,
+            but researchers are uncertain how the trait evolved</figcaption>
     </figure>
-
-    <h2><img src=https://static01.nyt.com/images/2018/06/12/multimedia/author-carl-zimmer/author-carl-zimmer-thumbLarge.png>By
-            <a href="https://www.nytimes.com/by/carl-zimmer">
-        Carl Zimmer</a></h2>
-
+    <p class="class2">Carl Zimmer </p>
     <p>Nov. 30, 2018</p>
-
-    <p>To demonstrate how smart an octopus can be, Piero Amodio points to a YouTube video. It shows an octopus pulling
-        two halves of a coconut shell together to hide inside. Later the animal stacks the shells together like nesting
-        bowls — and carts them away.</p>
-
-    <p>“It suggests the octopus is carrying these tools around because it has some understanding they may be useful in
-        the future,” said Mr. Amodio, a graduate student studying animal intelligence at the University of Cambridge in
-        Britain.</p>
-
+    <p>To demonstrate how smart an octopus can be,
+        Piero Amodio points to a YouTube video. It shows an octopus pulling two halves of a coconut shell together to
+        hide inside. Later the animal stacks the shells together like nesting bowls — and carts them away.</p>
     <iframe width="560" height="315" src="https://www.youtube.com/embed/1DoWdHOtlrk" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
         allowfullscreen></iframe>
-
+    <aside>
+        <blockquote>
+            “It suggests the octopus is carrying these tools around because it has some understanding they may be
+            useful in the future,” said Mr. Amodio,
+        </blockquote>
+    </aside>
+    <p>"It suggests the octopus is carrying these tools around because it has some understanding they may be
+        useful in the future,” said Mr. Amodio,a graduate student studying animal intelligence at the University of
```

Comment:
Missing a space in the text:
```suggestion
        useful in the future,” said Mr. Amodio, a graduate student studying animal intelligence at the University of
```